inlang/source-code/ide-extension/src/utilities/project/project.ts

```diff
@@ -103,23 +103,6 @@ export async function handleTreeSelection(args: {
 			_import: _import(normalizePath(args.workspaceFolder.uri.fsPath)),
 		})
 
-		if (inlangProject.id) {
-			telemetry.groupIdentify({
-				groupType: "project",
-				groupKey: inlangProject.id,
-			})
-		}
-
```

Comment:
the group identify is not needed anymore because the project group is automatically send along any capture